tests/unit/BusinessContactInfo.spec.ts

```diff
@@ -54,19 +53,17 @@ function getLastEvent (wrapper: Wrapper<BusinessContactInfo>, name: string): any
  * @returns a Wrapper<BusinessContactInfo> object with the given parameters.
  */
 function createComponent (
-  email: string = '',
-  confirmEmail: string = '',
-  phone: string = '',
-  extension?: number,
-  isEditing: boolean = true
+  email = '',
+  confirmEmail = '',
+  phone = '',
+  extension: number = null,
```

Comment:
Small semantic fix. Now extension is null (instead of undefined) if omitted as a formal parameter.